roxie/ccd/ccdserver.cpp

```diff
@@ -4174,7 +4174,7 @@ class CRemoteResultAdaptor : implements IEngineRowStream, implements IFinalRoxie
                             continue;
                         activity.queryLogCtx().CTXLOG("Input has not been acknowledged for %u ms - retry required?", timeout);
```

Comment:
We also need to prevent a lot of these getting logged.  We don't want/need many thousands of these log lines during high loads.

Reply:
Perhaps like lastRetryCheck we could track with a new lastLoggedTime and only log these every 5 sec or so (with a count?) ?